mlrun/datastore/sources.py

```diff
@@ -875,21 +873,45 @@ def __init__(
         collection_name: str = None,
         spark_options: dict = None,
     ):
-        key = "MONGO_CONNECTION_STRING"
-        connection_string = connection_string or os.getenv(key)
+        """
+        Reads MongoDB as input source for a flow.
+
+        example::
+             connection_string = "???"
+             query = {age: {"$gt: 5}}
+             MongoDBSource(connection_string=connection_string, collection_name="coll",
+             db_name="my_dataset", chunksize=5, query=query)
+
+        :param name:  source name
+        :param query: dictionary query for mongodb
+        :param chunksize: number of rows per chunk (default large single chunk)
+        :param key_field: the column to be used as the key for events. Can be a list of keys.
+        :param time_field: the column to be parsed as the timestamp for events. Defaults to None
+        :param start_time: filters out data before this time
+        :param end_time: filters out data after this time
+        :param schedule: string to configure scheduling of the ingestion job. For example '*/30 * * * *' will
+             cause the job to run every 30 minutes
+        :param db_name: the name of the database to access
+        :param connection_string: your mongodb connection string
+        :param collection_name: the name of the collection to access,
+                                        from the current database
+        :param spark_options: additional spark read options
+        """
+
+        MONGO_CONNECTION_STRING_ENV_VAR = "MONGO_CONNECTION_STRING"
```

Comment:
move to be at the class level, all caps is marking a static variable, if its usage is only internal of the class then it should be at thee class level, if its only for the developers use, it should start with a `_` to be marked as protected.